client/.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: [
+    'plugin:react/recommended',
```

Comment:
does the rule of using `"` vs `'` not apply in this file?

Reply:
good question, looks like it's not picking it up, I'll change it manually